src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/store/DummyGraph.java

```diff
@@ -125,9 +136,9 @@ public void construct() {
     hotNodeClusterRca.addTag(TAG_LOCUS, LOCUS_MASTER_NODE);
     hotNodeClusterRca.addAllUpstreams(Collections.singletonList(hotJVMNodeRca));
 
-    // constructResourceHeatMapGraph();
+    constructResourceHeatMapGraph();
```

Comment:
If we are doing this, we might want to add it to muted RCA list. We don't want this enabled by default, right ?